CRM/Contribute/WorkflowMessage/ContributionTrait.php

```diff
@@ -36,4 +43,17 @@ public function setContribution(array $contribution): self {
     return $this;
   }
 
+  /**
+   * Extra variables to be exported to smarty based on being calculated.
+   *
+   * We export isShowTax to denote whether invoicing is enabled but
+   * hopefully at some point we will separate the assumption that invoicing
+   * and tax are a package.
+   *
+   * @param array $export
+   */
+  protected function exportExtraTplParams(array &$export): void {
```

Comment:
It's not blocking but I'm wondering if there's some way to phpdoc these to avoid somebody going "I grepped and couldn't find anywhere it's used so let's remove it". You have to hunt to find that there's reflection being used to find functions starting with `exportExtra`.